.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.mjs]
+max_line_length = 999
```

Comment:
I know this looks a lot, but it's mainly to prevent wrapping inline SVG paths. We can still be sensible with line-lengths elsewhere.